### ISSUE TITLE
Scripting API command loadFileInteractiveQualified() fails silently

### ISSUE THREAD

**Issue by darth** [OpenModelica/OpenModelica#9114]
### Description

Scripting API command `loadFileInteractiveQualified()` fails silently. 

### Steps to Reproduce
```
loadFileInteractiveQualified("Test.mo"); getErrorString()
```

File with `Test.mo` [Test.zip](https://github.com/OpenModelica/OpenModelica/files/8920607/Test.zip). Could any other valid model file.

### Expected Behavior

Expected reply in case of success is the list loaded models and empty error string.
```
{Test}
""
```

### Version and OS

 - OpenModelica Version: 1.19.0
 - OS: Windows 10, 64 bit

### Additional Context

Looking at the code, it seems that the issue is because of the mismatch of the "signatures" in [OMCompiler/Compiler/Script/CevalScript.mo](https://github.com/OpenModelica/OpenModelica/blob/1b635573bf0f9557562cb50d94b26573b4b3f3c3/OMCompiler/Compiler/Script/CevalScript.mo#L631) and [OMCompiler/Compiler/NFFrontend/NFModelicaBuiltin.mo](https://github.com/OpenModelica/OpenModelica/blob/1b635573bf0f9557562cb50d94b26573b4b3f3c3/OMCompiler/Compiler/NFFrontEnd/NFModelicaBuiltin.mo#L1459).

**Comment by perost** [comment]
Thanks, seems #6784 added the new arguments to the wrong function in NFModelicaBuiltin. I updated the test case to use the new frontend so it doesn't break again.

**Comment by casella** [comment]
Thanks @darth for the report!

**Comment by perost** [comment]
I forgot I wanted to cherry-pick the fix for the 1.19 maintenance branch too so it's included in the next release, I made a PR for it now.

**Comment by casella** [comment]
Works fine in 1.19.2-dev.beta2
